src/lib/importer/hoyoLabFormatParser.tsx

```diff
@@ -0,0 +1,334 @@
+        },
+        substats: substats,
+        verified: false,
+      })
+    }
+    if (character.id == 8002) {
+      output.metadata.trailblazer = 'Stelle'
+      output.metadata.current_trailblazer_path = 'Destruction'
+    }
+    if (character.id == 8004) {
+      output.metadata.trailblazer = 'Stelle'
+      output.metadata.current_trailblazer_path = 'Preservation'
+    }
+    if (character.id == 8006) {
+      output.metadata.trailblazer = 'Stelle'
+      output.metadata.current_trailblazer_path = 'Harmony'
+    }
+    if (character.id == 8001) {
+      output.metadata.trailblazer = 'Caelus'
+      output.metadata.current_trailblazer_path = 'Destruction'
+    }
+    if (character.id == 8003) {
+      output.metadata.trailblazer = 'Caelus'
+      output.metadata.current_trailblazer_path = 'Preservation'
+    }
+    if (character.id == 8005) {
+      output.metadata.trailblazer = 'Caelus'
+      output.metadata.current_trailblazer_path = 'Harmony'
+    }
+  }
+  output.relics.map((r) => RelicAugmenter.augment(r))
+  return output
+}
+
+function readValue(value: string) {
+  if (value.endsWith('%')) {
+    return Utils.precisionRound(parseFloat(value.slice(0, value.length - 1)))
+  }
+  return Utils.precisionRound(parseFloat(value))
+}
+
+function getStat(id: number) {
+  switch (id) {
+    case 27:
+      return Constants.Stats.HP
+    case 32:
+      return Constants.Stats.HP_P
+    case 29:
+      return Constants.Stats.ATK
+    case 33:
+      return Constants.Stats.ATK_P
+    case 31:
+      return Constants.Stats.DEF
+    case 34:
+      return Constants.Stats.DEF_P
+    case 51:
+      return Constants.Stats.SPD
+    case 52:
+      return Constants.Stats.CR
+    case 53:
+      return Constants.Stats.CD
+    case 56:
+      return Constants.Stats.EHR
+    case 57:
+      return Constants.Stats.RES
+    case 59:
+      return Constants.Stats.BE
+    case 54:
+      return Constants.Stats.ERR
+    case 55:
+      return Constants.Stats.OHB
+    case 12:
+      return Constants.Stats.Physical_DMG
+    case 14:
+      return Constants.Stats.Fire_DMG
+    case 16:
+      return Constants.Stats.Ice_DMG
+    case 18:
+      return Constants.Stats.Lightning_DMG
+    case 20:
+      return Constants.Stats.Wind_DMG
+    case 22:
+      return Constants.Stats.Quantum_DMG
+    case 24:
+      return Constants.Stats.Imaginary_DMG
+    default:
+      return ''
+  }
+}
+
+function getSlot(id: number) {
+  switch (id) {
+    case 1:
+      return Parts.Head
+    case 2:
+      return Parts.Hands
+    case 3:
+      return Parts.Body
+    case 4:
+      return Parts.Feet
+    case 5:
+      return Parts.PlanarSphere
+    case 6:
+      return Parts.LinkRope
+    default:
+      return ''
+  }
+}
+
+function getSet(id: number) {
+  let set = ''
+  switch (Math.floor((id % 10000) / 10)) { // Set
+    case 101:
+      set = Sets.PasserbyOfWanderingCloud
+      break
+    case 102:
+      set = Sets.MusketeerOfWildWheat
+      break
```

Comment:
We should probably parse the IDs out of src/data/relic_sets.json and then join it with Constants.Sets instead of hardcoding, otherwise would have to remember to update the scanner import every time there's a new set